cavityDisplayGUI.py

```diff
@@ -85,8 +84,9 @@ def __init__(self, parent=None, args=None):
 
                 cryomoduleObject = linacObject.cryomodules[str(cmButton.text())]
                 cavityWidgetList: List[CavityWidget] = cmTemplateRepeater.findChildren(CavityWidget)
+                rfStatusBarList = cmTemplateRepeater.findChildren(PyDMDrawingLine)
 
-                for cavityWidget in cavityWidgetList:
+                for cavityWidget, rfStatusBar in zip(cavityWidgetList, rfStatusBarList):
```

Comment:
Question: did you verify that the bars are "in order" of cavity? Because it could be worth double checking by doing our favorite accessible name thing again.

Reply:
I think it prints in order:D Here's what I did to test it:

1. In designer, I gave the status bar an accessibleName of `ACCL:${linac}:${cryoNum}${cavityNumber}0:RFSTATE` aka the pv for the associated status bar.
2. In cavityDisplayGUI.py in the forloop for sifting through cavityWidgets (the forloop referenced in this comment actually), I added a print statement `print(rfStatusBar.accessibleName())` and everything printed in order! As in, I got print out in order starting from L0B cryo 01 cavities 1 -8, cryo 02, cavities 1-8, etc for each linac.
3. I also added a hover text for the status bar that reads out the accessible name (aka the pv) and on my mac, they look to match up

Would we hypothetically want the hover text for a status bar or nah? And are there any other troubleshooting things I should try to see if the status bars are actually in order?

Reply:
Questions/comments/concerns?